static/docs/command-reference/version.md

```diff
@@ -29,8 +29,8 @@ system/environment:
 
 <!-- Separate MD quote: -->
 
-> **Note** that if you've installed dvc using pip, you will need to install
-> `psutil` by yourself with `pip install psutil` in order for `dvc version` to
+> Note that if you've installed DVC using `pip`, you will need to install
+> `psutil` manually with `pip install psutil` in order for `dvc version` to
 > report file system information. Please see the original
 > [issue on GitHub](https://github.com/iterative/dvc/issues/2284) for more info.
```

Comment:
Same Q guys (Ivan, @efiop) still a valid note? And/or any updated on the issue? I see it's still open. Just not sure we want these kind of notes about GH issues hardcoded into the docs.

Reply:
I think it's fine just in this particular case until we resolve it - it might take some time.

Reply:
OK but we should have notes in those issues to remind people to update the docs when resolved. I'll add them... (e.g. https://github.com/iterative/dvc/issues/2284#issuecomment-564295197)

Reply:
👍